cfn-resources/maintenance-window/README.md

```diff
@@ -0,0 +1,60 @@
+# Mongodb::Atlas::MaintenanceWindow
+
+## Description
+The MaintenanceWindow resource provides access to retrieve or update the current Atlas project maintenance window.
+
+## Attributes & Parameters
+
+Please consult the [Resource Docs](docs/README.md)
+
+## Local Testing
+
+The local tests are integrated with the AWS `sam local` and `cfn invoke` tooling features:
+
+```
+sam local start-lambda --skip-pull-image
+```
+then in another shell:
+```bash
+repo_root=$(git rev-parse --show-toplevel)
+source <(${repo_root}/quickstart-mongodb-atlas/scripts/export-mongocli-config.py)
+cd ${repo_root}/cfn-resources/project
+./test/project.create-sample-cfn-request.sh YourProjectName > test.request.json 
+echo "Sample request:"
+cat test.request.json
+cfn invoke CREATE test.request.json 
+cfn invoke DELETE test.request.json 
+```
+
```

Comment:
Missing example (include link). Also if template json file doesn't exist please create it. 

https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples

Reply:
yes! i forgot that in the other repository, i added the example file and some descriptions tooh